.github/dependabot.yml

```diff
@@ -1,11 +1,26 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
 version: 2
 updates:
-  - package-ecosystem: 'npm'
-    directory: '/site'
+  - package-ecosystem: npm
+    directory: /site
     schedule:
       interval: 'monthly'
+    groups:
+      gatsby:
+        patterns:
+          - '*gatsby*/*'
+          - '*/*gatsby*'
+          - 'gatsby*'
```

Comment:
I believe only "gatsby*" is the working entry here.  https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups